.github/workflows/ci.yml

```diff
@@ -189,7 +189,7 @@ jobs:
         # uses: obi1kenobi/cargo-semver-checks-action@v2
         uses: n0-computer/cargo-semver-checks-action@feat-baseline
         with:
-          package: iroh, iroh-base, iroh-dns-server, iroh-net-bench, iroh-relay, iroh-net-report
+          package: iroh, iroh-base, iroh-dns-server, iroh-bench, iroh-relay, iroh-net-report
```

Comment:
so iroh-net-report does not get renamed to iroh-report for now?

Reply:
don't know if we want to do that, but that's seperate from this PR